### DISCUSSION TITLE
How to display an SVG graphic?

### DISCUSSION THREAD

**Discussion by BytePetter** [AvaloniaUI/Avalonia#11995] [Q&A]
Question is in the topic.
I have a simple svg-file that I need to display, but there is no way to do it in Avalonia, it seems.

Help would be appreciated.

**Comment by wieslawsoltes** [comment]
https://github.com/wieslawsoltes/Svg.Skia

**Reply by BytePetter** [reply]
Could you please elaborate a little bit? I already have this package installed, but still aren't able to make an SVG appear.

**Reply by wieslawsoltes** [reply]
Check Readme file and sample applications.
